src/FINDOLOGIC/Export/Data/Attribute.php

```diff
@@ -13,18 +14,22 @@ class Attribute implements Serializable
 
     public function __construct($key, $values = array())
     {
-        $this->key = $key;
-        $this->values = $values;
+        $this->key = DataHelper::emptyValueCheck($key);
```

Comment:
[Suppress](https://phpmd.org/documentation/suppress-warnings.html) the Code Climate warning for using static access here, because this is a legitimate use case.

Reply:
Done